avocado/core/remote/runner.py

```diff
@@ -325,8 +329,8 @@ class VMTestRunner(RemoteTestRunner):
     Test runner to run tests using libvirt domain
     """
 
-    def __init__(self, job, result_proxy):
-        super(VMTestRunner, self).__init__(job, result_proxy)
+    def __init__(self, job, result_proxy, result):
```

Comment:
Because you talked about this before, I understand this patch, but I think it'd be clearer if you introduced it together with the patch which makes the actual use of it (instead of both, result_proxy and result as storage). Right now this patch looks strange to me. (unless the next patch implements what I just mentioned).